bob-common/src/configs/node.rs

```diff
@@ -310,6 +312,14 @@ impl Pearl {
         true
     }
 
+    pub fn disks_events_logfile(&self) -> &str {
+        &self.disks_events_logfile
+    }
+
+    fn default_disks_events_logfile() -> String {
+        "/tmp/bob_events.csv".to_owned()
```

Comment:
IMO we should try to get OS tmp dir here, and fallback to /tmp in case of no env set